tools/webpack.config.prod.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var srcPath = root('src');
+var componentsPath = root('src/components');
+var sassPath = root('src/sass');
+var docsPath = root('src/docs');
+var fontsPath = root('src/fonts');
+
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
+  },
+  plugins: [
+    new webpack.NoErrorsPlugin(),
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.optimize.DedupePlugin(), 
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('production'),
```

Comment:
Do you mean the `DefinePlugin` section?
I am not sure whether there will be any env variables in the app code at all